crates/miden-objects/src/account/file.rs

```diff
@@ -77,6 +78,12 @@ impl Serializable for AccountFile {
 
 impl Deserializable for AccountFile {
     fn read_from<R: ByteReader>(source: &mut R) -> Result<Self, DeserializationError> {
+        let magic_value = source.read_string(4)?;
+        if magic_value != "acct" {
+            return Err(DeserializationError::InvalidValue(format!(
+                "Invalid account file marker: {magic_value}"
```

Comment:
nit: I would change "Invalid" not to be capitalized - i.e., "invalid". We should probably do the same for `NoteFile`.